lib/jerboa/format/body/attribute/dont_fragment.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Jerboa.Format.Body.Attribute.DontFragment do
```

Comment:
Looks quite complicated for an attribute that is "empty" ;) but it ofc follows the design we have